recipes/dcmtk/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "DCMTK" %}
+{% set version = "3.6.7" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://dicom.offis.de/download/dcmtk/dcmtk{{ version | replace('.','') }}/dcmtk-{{ version }}.tar.gz
+  sha256: 7c58298e3e8d60232ee6fc8408cfadd14463cc11a3c4ca4c59af5988c7e9710a
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+  host:
+    - openjpeg
+    - libpng
+    - libsndfile
+    - libtiff
+    - libxml2
+    - openssl
+    - zlib
+  run:
+    - libpng
+    - libsndfile
+    - libtiff
+    - openjpeg
+    - libxml2
+    - openssl
+    - zlib
```

Comment:
```suggestion
```
The run exports are not always the same package. See warnings:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=602154&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&l=5805